### ISSUE TITLE
Bug: movetoworkspacesilent, special 

### ISSUE THREAD

**Issue by fedoranvar** [hyprwm/Hyprland#310]
Good day to you!

i have this in config:
```bash
bind=SUPERSHIFT,bar,movetoworkspacesilent,special
bind=SUPER,backslash,togglespecialworkspace
```

when i moving window from named workspace to `special`, my current monitor switches named workspace to numbered workspace (e.g. with ID 1)

thank you!

**Comment by vaxerski** [comment]
fixed in 239aa2cec4373f45ca55be74d5768b42abf1e749

**Comment by fedoranvar** [comment]
it works good!

Thank you!
